xcode/Subconscious/Shared/Components/Common/Forms/ValidatedTextField.swift

```diff
@@ -7,21 +7,21 @@
 
 import SwiftUI
 import Combine
+import ObservableStore
 
-/// A text field that comes with help text and a validation flag
-struct ValidatedTextField: View {
+struct ValidatedFormField<T: Equatable>: View {
+    @State private var innerText: String = ""
+    @FocusState private var focused: Bool
```

Comment:
I like the way we've wrapped this up. It's unfortunate that we have to replay changes, but the way we're doing it makes it an invisible implementation detail of the view. 👍